### PR TITLE
how-to/scale: add necessary actions on the target machines

### PR DESCRIPTION
**Pull request by toutdesuite**
### What is changed, added or deleted? (Required)

add Install the NTP service on the target machines
add Configure the SSH mutual trust and sudo rules on the Control Machine

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v4.0 changes for now)
- [x] v3.1 (TiDB 3.1 versions)
- [x] v3.0 (TiDB 3.0 versions)
- [x] v2.1 (TiDB 2.1 versions)

**If you select two or more versions from above**, to trigger the bot to cherry-pick this PR to your desired release version branch(es), you **must** add corresponding labels such as **needs-cherry-pick-3.1**, **needs-cherry-pick-3.0**, and **needs-cherry-pick-2.1**.

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/2359 
- Other reference link(s): https://github.com/pingcap/docs-cn/pull/2324

### CODE REVIEW

**Comment by toutdesuite** [comment]
@superlzs0476 PTAL.

File: how-to/scale/with-ansible.md [hunk 1]
```diff
@@ -30,45 +30,47 @@ Assume that the topology is as follows:
...
-    172.16.10.101
-    172.16.10.102
-
-    [monitoring_servers]
-    172.16.10.3
-
-    [grafana_servers]
-    172.16.10.3
-    ```
+1. Edit the `inventory.ini` file and the `hosts.ini` file, and append the node information:
```
Comment (TomShawn) [inline how-to/scale/with-ansible.md]:
```suggestion
1. Edit the `inventory.ini` file and the `hosts.ini` file, and append the node information.
```

File: how-to/scale/with-ansible.md [hunk 2]
```diff
@@ -86,11 +88,51 @@ For example, if you want to add two TiDB nodes (node101, node102) with the IP ad
...
+        172.16.10.8
+        172.16.10.9
+        172.16.10.101
+        172.16.10.102
+        [all:vars]
+        username = tidb
+        ntp_server = pool.ntp.org
+        ```
+
 2. Initialize the newly added node:
```
Comment (TomShawn) [inline how-to/scale/with-ansible.md]:
```suggestion
2. Initialize the newly added node.
```

**Comment by superlzs0476** [comment]
LGTM

**Pull request merged**